Cannot successfully perform 'npm install'

There appears to be a dependency that points to a non-existing branch:
https://github.com/colinbdclark/Flocking/blob/master/package.json#L36

Result of ``npm install``:

```
npm ERR! git rev-list -n1 gh-6: fatal: ambiguous argument 'gh-6': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 gh-6: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 gh-6: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 gh-6:
npm ERR! git rev-list -n1 gh-6: fatal: ambiguous argument 'gh-6': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 gh-6: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 gh-6: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 gh-6:
<snip, snip>
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
```

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
   "dependencies": {
     "jquery": "2.1.4",
     "infusion": "2.0.0-dev.20160519T222603Z.754d2c6",
-    "codemirror-infusion": "colinbdclark/codemirror-infusion#gh-6",
+    "codemirror-infusion": "2.1.0",
     "jsplumb": "1.7.9",
     "dagre": "0.7.4",
     "normalize.css": "3.0.3"

